tldw_Server_API/tests/StudySuggestions/test_flashcard_review_sessions.py

```
@@ -0,0 +1,362 @@
+        review_row = db.get_latest_flashcard_review(card_uuid)
+        sessions = db.list_flashcard_review_sessions(deck_id=deck_id)
+
+        assert updated["review_session_id"] == session["id"]  # nosec B101
+        assert review_row["review_session_id"] == session["id"]  # nosec B101
+        assert sessions[0]["id"] == session["id"]  # nosec B101
+    finally:
+        try:
+            db.close_connection()
+            if db.backend_type.name == "POSTGRESQL":
+                db.backend.get_pool().close_all()
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)